src/tools/prun/prun.1.md

```diff
@@ -443,34 +281,39 @@ The parser for the `-x` option is not very sophisticated; it does not
 even understand quoted values. Users are advised to set variables in the
 environment, and then use `-x` to export (not define) them.
 
-Setting MCA parameters:
+## MCA Parameters
 
-`-gpmca, --gpmca <key> <value>`
+Setting MCA parameters take a few different forms depending the target
+project for the parameter. For example, MCA parameters targeting OpenPMIx
+will contain the string `pmix` in their name, and MCA parameters targeting
+PRTE will contain the string `prte` in their name. See the "MCA" section,
+below, for finer details on the MCA.
 
-:   Pass global MCA parameters that are applicable to all contexts.
+`--gpmixmca <key> <value>`
+
+:   Pass global PMIx MCA parameters that are applicable to all contexts.
```

Comment:
What is the difference between global PMIx MCA parameters and what's passed with --pmixmca, partly because I'm unfamiliar with the context concept. Well, reading forward thru the page I see discussion of app contexts primarily for MPMD, so I think clarifies global (all contexts) vs settings for a specific app context, where I'm guessing the specific options would be in a context file

Reply:
I clarified 'contexts' here a bit to help:
 > :   Pass global PMIx MCA parameters that are applicable to all application contexts. `<key>` is the parameter name; `<value>` is the parameter value.